app/controllers/observations/search_controller.rb

```diff
@@ -27,7 +27,7 @@ def permitted_search_params
         region: :region_with_in_box_fields,
         in_box: :in_box_fields,
         has_specimen: :select_nil_boolean,
-        has_sequences: :select_nil_yes, # ignores false
+        has_sequences: :select_nil_boolean,
```

Comment:
Guided tour for @JoeCohen

In these lines in the `Observations::SearchController`, we are
- listing which `Query::Observations` attributes are available in the form
- explicitly defining the **_UI helper method_** to call for the attribute/param. This could be done automatically, now that i think about it, but we still need to list the available attributes, since not all are available.